### PR TITLE
Punctuation/grammer fixes in rpcwallet.cpp

### PR DESCRIPTION
**Pull request by stevendlander**
Standardizing punctuation on CLI output and also including a few fixes for grammer.  This PR is for text only changes and includes no code edits.

### CODE REVIEW

**Comment by MeshCollider** [comment]
NACK, disagree with the addition of full stops to every line and most of the changes in capitalization. Only a couple of the changes are valid spelling fixes, prefer if they were isolated from the rest

**Comment by stevendlander** [comment]
Isolated spelling fixes and reverted all but one capitalization.  Will revert that too if needed, thanks.

File: src/wallet/rpcwallet.cpp [hunk 1]
```diff
@@ -719,7 +719,7 @@ UniValue getbalance(const JSONRPCRequest& request)
             "\nExamples:\n"
             "\nThe total amount in the wallet\n"
             + HelpExampleCli("getbalance", "") +
-            "\nThe total amount in the wallet at least 5 blocks confirmed\n"
+            "\nThe total amount in the wallet at least 6 blocks confirmed\n"
```
Comment (jonasschnelli) [inline src/wallet/rpcwallet.cpp]: Ideally we include `wallet.h`'s `DEFAULT_TX_CONFIRM_TARGET` here somehow.

Comment (stevendlander) [inline src/wallet/rpcwallet.cpp]: Should I code up a solution to that in this PR?

**Comment by stevendlander** [comment]
I'll tidy up the places where the const char needs to be an unsigned int.

**Comment by stevendlander** [comment]
Open to suggestions on a better approach for string-ifying `DEFAULT_TX_CONFIRM_TARGET`.  I would imagine importing `<string>` just to do that is not ideal.

File: src/wallet/rpcwallet.cpp [hunk 2]
```diff
@@ -587,10 +588,10 @@ UniValue getreceivedbyaddress(const JSONRPCRequest& request)
             + HelpExampleCli("getreceivedbyaddress", "\"1D1ZrZNe3JUo7ZycKEYQQiQAWd9y54F4XX\"") +
             "\nThe amount including unconfirmed transactions, zero confirmations\n"
             + HelpExampleCli("getreceivedbyaddress", "\"1D1ZrZNe3JUo7ZycKEYQQiQAWd9y54F4XX\" 0") +
-            "\nThe amount with at least 6 confirmation, very safe\n"
-            + HelpExampleCli("getreceivedbyaddress", "\"1D1ZrZNe3JUo7ZycKEYQQiQAWd9y54F4XX\" 6") +
+            "\nThe amount with at least " + std::to_string(DEFAULT_TX_CONFIRM_TARGET) + " confirmation, very safe\n"
```
Comment (TheBlueMatt) [inline src/wallet/rpcwallet.cpp]: lol, lets remove the "very safe" doge-speak...

**Comment by morcos** [comment]
NACK.  

I don't like tying the DEFAULT_TX_CONFIRM_TARGET to the concept of how many confirmations we might want to wait before considering a transaction to have low chance of being re-orged out.  These are unrelated concepts that just both happen to use the number 6 right now.

**Comment by jtimon** [comment]
What about adding a different constant?

File: src/wallet/rpcwallet.cpp (hunk 1 above)
Comment (morcos) [inline src/wallet/rpcwallet.cpp]: No. It's a different concept

**Comment by stevendlander** [comment]
Updated to address @morcos feedback.

**Comment by FilmCoder** [comment]
Very grammar, much pull request, very moon merge wow.

**Comment by jtimon** [comment]
utACK e439bebdc03ea42bb76c4fce9edbaf0470ddd3d9
Please squash into one commit.

**Comment by TheBlueMatt** [comment]
utACK a5ecaf1490be8d01ac6806785157230d30f77061, we should take this for 15.

**Pull request merged**